introduction-en.md

```diff
@@ -1,6 +1,7 @@
 # Introduction: Open invitation for market parties to join the OpenZaak community
 
-<div class="navoz"><a href="https://hackmd.io/b6mszG6HRjyDGRCi72_y2w">Home</a> | Introduction | <a href="https://hackmd.io/OFGiBKmLSh6ug7Gpv3Hbzw">Our proposals and questions</a> | <a href="https://hackmd.io/AQOmgrpSQIeMVC4OhFxKDA">How to respond</a></div>
+| <a href="https://hackmd.io/b6mszG6HRjyDGRCi72_y2w">Home</a> | Introduction | <a href="https://hackmd.io/OFGiBKmLSh6ug7Gpv3Hbzw">Our proposals and questions</a> | <a href="https://hackmd.io/AQOmgrpSQIeMVC4OhFxKDA">How to respond</a> |
```

Comment:
| <a href="https://hackmd.io/b6mszG6HRjyDGRCi72_y2w">Home</a> | Introduction | <a href="https://hackmd.io/OFGiBKmLSh6ug7Gpv3Hbzw">Our proposals and questions</a> | <a href="https://hackmd.io/AQOmgrpSQIeMVC4OhFxKDA">How to respond</a> |

Still links to hackmd?